packages/manager/modules/iplb/src/onboarding/constants.js

```diff
@@ -91,6 +91,7 @@ export const CTAS = {
   FR: 'https://www.ovhcloud.com/fr/network/load-balancer/',
   WE: 'https://www.ovhcloud.com/en-ie/network/load-balancer/',
   WS: 'https://www.ovhcloud.com/es/network/load-balancer/',
+  US: 'https://us.ovhcloud.com/network/load-balancer/',
```

Comment:
this gives a 404

Reply:
yes its not live yet. this will be ready before going to prod.